src/fu-io-channel.c

```diff
@@ -16,6 +16,7 @@
 #include <poll.h>
 #endif
 #include <string.h>
+#include <sys/stat.h>
```

Comment:
Unneeded include, right?

Reply:
Nah, actually needed for the S_IRWXU define. I can split out this hunk as another commit if that's better.